google-cloud-spanner/src/main/java/com/google/cloud/spanner/MultiplexedSessionDatabaseClient.java

```diff
@@ -267,6 +299,59 @@ private void maybeMarkUnimplemented(Throwable t) {
     }
   }
 
+  private void maybeMarkUnimplementedForRW(SpannerException spannerException) {
+    if (spannerException.getErrorCode() == ErrorCode.UNIMPLEMENTED
+    // && spannerException.getCause().getMessage().contains("Transaction type read_write not
+    // supported with multiplexed sessions")
```

Comment:
nit: either remove commented code or re-enable it (see also comment above)

Reply:
Added error message verification.